contracts/SpokePoolV3Periphery.sol

```diff
@@ -77,216 +82,99 @@ abstract contract SwapAndBridgeBase is Lockable, MultiCaller {
     error MinimumExpectedInputAmount();
     error LeftoverSrcTokens();
     error InvalidFunctionSelector();
+    error ContractInitialized();
+    error InvalidMsgValue();
+    error InvalidSpokePool();
+    error InvalidSwapToken();
 
     /**
      * @notice Construct a new SwapAndBridgeBase contract.
-     * @param _spokePool Address of the SpokePool contract that we'll submit deposits to.
-     * @param _exchange Address of the exchange where tokens will be swapped.
      * @param _allowedSelectors Function selectors that are allowed to be called on the exchange.
      */
-    constructor(
-        V3SpokePoolInterface _spokePool,
-        WETH9Interface _wrappedNativeToken,
-        address _exchange,
-        bytes4[] memory _allowedSelectors
-    ) {
-        spokePool = _spokePool;
-        exchange = _exchange;
-        wrappedNativeToken = _wrappedNativeToken;
+    constructor(bytes4[] memory _allowedSelectors) {
         for (uint256 i = 0; i < _allowedSelectors.length; i++) {
             allowedSelectors[_allowedSelectors[i]] = true;
         }
     }
 
-    // This contract supports two variants of swap and bridge, one that allows one token and another that allows the caller to pass them in.
```

Comment:
The code to these functions were not changed, (although since it is a single contract, their visibility was switched to `private` for gas savings), but they were moved to the bottom of the contract to follow the Solidity style guide.